style.css

```diff
@@ -1,554 +1,1111 @@
   flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  height: 10rem;
-  padding-top: 4rem;
-  width: 21.6rem;
-  left: 0;
-  top: 0;
-  color: var(--txtcolor2);
+  background-color: #fff;
 }
 
-.headtext {
-  margin: 0;
-  padding: 0;
-  border: none;
-}
-
-.work > .first-box > .headtext > h2 {
-  font-size: 2.2rem;
+.recent-work h2 {
+  font-size: 40px;
+  font-weight: 400;
   font-family: 'Crete Round', serif;
-  font-weight: 500;
-  justify-items: center;
-  padding: 5rem 1rem 2rem 0;
-  border-bottom: 1.5px solid var(--line);
-  width: 18rem;
-}
-
-.ttwork {
-  padding: 0%;
-  margin: 0%;
+  width: 100%;
+  line-height: 2rem;
+  height: 83px;
+  text-align: center;
+  margin-bottom: 10px;
+  color: #1c1a19;
 }
 
-.carttt {
-  justify-self: center;
-  padding: 0;
-  margin: 0;
+.recent-work hr {
   width: auto;
+  margin-top: 4px;
+  margin-bottom: 20px;
+  color: #212121;
 }
 
-div.first-box {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-  padding: 0;
-  width: 22rem;
-  margin-left: 0%;
-  margin-bottom: 2rem;
+.recent-work img {
+  width: auto;
+  margin-bottom: 15px;
 }
 
-div.imgsection {
-  padding: 0;
-  margin: 0;
-  width: 87%;
+.recent-work h3,
+.text {
+  margin-bottom: 7px;
+  color: #091e42;
+  font-size: 32px;
+  line-height: 2rem;
+  font-weight: 400;
+  font-family: 'Crete Round', serif;
+  width: auto;
 }
 
-.work-images {
-  padding-top: 3rem;
-  position: relative;
-  max-width: 19.1rem;
+.recent-work p.text {
+  line-height: 1.5rem;
+  font-size: 16px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
 }
 
-.work > .first-box > .imgsection:nth-child() {
+section.recent-work ul {
   display: flex;
-  padding-bottom: 2rem;
-  padding-right: 2rem;
-  justify-content: center;
+  width: 90%;
+  justify-content: space-between;
+  height: 40px;
+
+  /* margin-bottom: 5px; */
 }
 
-.about > .listing > ul > li > h3 {
-  display: flex;
-  flex-direction: row;
-  justify-content: center;
+.recent-work li {
+  border: 1px solid #8993a4;
   align-items: center;
-  font-size: 32px;
-  font-weight: 700;
-  line-height: 44px;
-  letter-spacing: 0;
-  text-align: center;
+  padding: 10px 12px;
+  list-style: none;
 }
 
-.listing {
-  border-top: 2px solid var(--txtcolor);
-  margin-top: 11rem;
-  padding-top: 8rem;
+.recent-work button {
+  width: 40%;
+  height: 40px;
+  padding: 12px;
+  background: #ff6b00;
+  margin: 12px 0;
+  color: #fff;
+  text-align: center;
+  text-decoration: none;
+  border: 0;
+  margin-bottom: 3px;
 }
 
-.about > .listing > ul > li > a {
-  text-decoration: none;
-  color: var(--txtcolor);
-  font-size: 22px;
-  font-weight: 400;
-  line-height: 43px;
-  letter-spacing: 0;
-  text-align: center;
+/* styling the project or work section */
+
+.work-container {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 386px;
+  padding: 10% 24px 35% 24px;
+  height: auto;
+  gap: 2%;
```

Comment:
- [ ] Kindly correct the section inside the red lines to be similar to the Figma template. 👇🏽 
- [ ] Please 🙏🏽 complete the suggestions from the last reviewer 👆🏽 before submitting your project again. Take in mind we only have 6 opportunities to approve it.

![image](https://user-images.githubusercontent.com/58642949/187038633-a5aed6f3-b352-45ad-a4ae-8287bc02fdd8.png)